components/CreateNewCampaign/MultipleFileUploadField.tsx

```diff
@@ -0,0 +1,121 @@
+import { Grid, Typography } from '@mui/material';
+import { useField } from 'formik';
+import React, { useCallback, useEffect, useState } from 'react';
+import { FileError, FileRejection, useDropzone } from 'react-dropzone';
+import SingleFileUploadWithProgress from './SingleFileUploadWithProgress';
+import { UploadError } from './UploadError';
+
+// import { makeStyles } from '@mui/styles';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+
+let currentId = 0;
+
+function getNewId() {
+  return ++currentId;
+}
```

Comment:
not good to have a global variable. if we really need an id, put it as a state in the component.